src/api/admin.ts

```diff
@@ -214,8 +220,8 @@ type DnaManifest = {
 }
 
 export type DnaBundle = {
-    manifest: DnaManifest,
-    resources: ResourceMap,
+    manifest: DnaManifest;
```

Comment:
Is this really supposed to be a ;?

Reply:
Yeap! Actually autoformatting with prettier fixes a lot of format issues that exist in this file at the moment. I could commit those if you wanted.

Reply:
That would be great because then it's standardized.  Thanks!